feat: remove `Control` and `ControlledConnection` (#164)
libp2p/rust-yamux: CHANGELOG.md

# 0.12.0 - unreleased

- Remove `Control` and `ControlledConnection`.

>   Users are encouraged to move to the `poll_` functions of `Connection`.

Reviewer: ```suggestion
  Users have to move to the `poll_` functions of `Connection`.
```

They don't really have an alternative, do they?
